scheduler/src/cook/scheduler/constraints.clj

```diff
@@ -92,6 +92,16 @@
   (let [previous-hosts (job->previous-hosts-to-avoid job)]
     (->novel-host-constraint job previous-hosts)))
 
+(defn job->disk-type-requested
```

Comment:
you're not passing in or using a job, right?
maybe just call the function disk-type-requested

and it doesn't seem like you need the let, just nested or:

(or (get (regexp-tools/match-based-on-pool-name (config/disk) pool-name :type-map) disk-type-requested)
        (or disk-type-from-user
                                (regexp-tools/match-based-on-pool-name (config/disk) pool-name :default-type)))

Reply:
`disk-type-requested` is used in 2 places, so that is why I opted to use a let block 

naming convention was consistent with `job->gpu-model-requested', but I am okay with changing it to just `disk-type-requested`

Reply:
done